code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -310,6 +332,30 @@
 		if(cleared)
 			to_chat(src, "--- [class] alarm in [A.name] has been cleared.")
 
+/mob/living/simple_animal/drone/proc/set_shy(newShy)
```

Comment:
is hacking/emagging drones still a thing? if so, that should make them no longer shy, and factory resetting them should reset their shy variable to its default value for their (drone) type.

Reply:
Yes, hacking a drone properly sets shy to false, and factory resetting (using a wrench on a drone) will call the unhacking procedure which will set shy to `initial(shy)`